pom.xml

```diff
@@ -749,48 +750,20 @@
           <target>${java.version}</target>
           <compilerArgs>
             <arg>-Xpkginfo:always</arg>
+            <arg>-XDcompilePolicy=simple</arg>
+            <arg>
+              -Xplugin:ErrorProne \
+              -XepExcludedPaths:.*[\\/]resources[\\/].*
+            </arg>
           </compilerArgs>
+          <annotationProcessorPaths>
+            <path>
+              <groupId>com.google.errorprone</groupId>
```

Comment:
please move error prone to `verify` phase.
https://github.com/google/error-prone/issues/985#issuecomment-378865047

it is better to let compile phase do only compilation.
Please investigate way to skip error-prone, as we do it in special profile 
https://github.com/checkstyle/checkstyle/blob/9b57025cb9f9be38790d3c09b7273e3ef5170b9a/pom.xml#L2134-L2138

Reply:
Done.

Reply:
Do we have a way to skip execution of errorprone  in `no-validations` profile?

Reply:
Yes, it is present in the PR